.github/workflows/fronts.yml

```diff
@@ -0,0 +1,40 @@
+name: DCR Fronts–Ophan Components
+on:
+  push:
+    branches: [main]
+
+  schedule:
+    # Every work day of the week at 08:08
+    - cron: "8 8 * * MON-FRI"
+
+  # Allows you to run this workflow manually from the Actions tab.
+  workflow_dispatch:
```

Comment:
Should the colon be removed? e.g just `workflow_dispatch`?

Reply:
This then = `workflow_dispatch: {}`, where Olly's suggestion is `workflow_dispatch: undefined`.